ReportEngine/ReportCommon/ProcurementPlan/PartOfProcurementPlan.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ReportCommon.ProcurementPlan
+{
+   public class PartOfProcurementPlan
+    {
+        public string ProcurementDesignAndProcess  { get; set; }
```

Comment:
Pas une bonne façon de nommer des classes avec des "And". Peux-tu trouver des alternatives ?